src/mcp/mcpConnectionManager.ts

```diff
@@ -42,13 +49,15 @@ To connect, choose a connection from MongoDB VSCode extensions's sidepanel - htt
     connectParams: MCPConnectParams,
   ): Promise<AnyConnectionState> {
     try {
+      const { connectionId, connectOptions, connectionString } =
+        this.overrideAppNameIfContainsVSCode(connectParams);
```

Comment:
nit: maybe name this something like `overridePresetAppName`, containing VSCode is an implementation detail